### ISSUE TITLE
Bold double barline misaligned with two staves

### ISSUE THREAD

**Issue by xaprb** [paulrosen/abcjs#190]
An ending bold double barline is misaligned when the second voice/staff is just rests. Test case:

```abc
X:1
M:6/8
L:1/8
%%score {(1) (2)}
V:1 clef=treble
V:2 clef=bass
K:G
% music
%
[V:1] c2c c2d|efg efg|f2d d2e|fga fga|gfe dcB|cBA ABc|d2D A2F|G3 G2|]
[V:2] z6 | z6 | z6 | z6 | z6 | z6 | z6 | z6 |]
```

Output:

<img width="793" alt="screen shot 2019-01-13 at 10 44 49 am" src="https://user-images.githubusercontent.com/279875/51087367-53924d00-1720-11e9-9f1a-cbe9dc51fffe.png">

Expected output is that the barlines encircled in red should align vertically.

**Comment by xaprb** [comment]
Looks like it might be my mistake: the final measure has only 5 8th notes in the first staff. Is the misalignment expected in that case?

Also, by inspecting the source of the site, I see that the current version of the abc editor is using version `abcjs_midi_5.1.2-min.js` so perhaps this is rendered differently in a more recent version? The latest is 5.6.2. I will check in that version.

**Comment by paulrosen** [comment]
Yes, that is an error case in the input string. There are, unfortunately, many ways to create inconsistencies in ABC. I thought about different options for how to display that, and decided to leave it like that - it should make you recount your timing.

It hasn't changed in the last few versions - the versions have been coming pretty quick and they are mostly small bug fixes or new features.
